content/en/blog/_posts/2024-08-21-cgroup-driver-from-cri.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: blog
+title: "Configuring Kubernetes just got easier: cgroup driver from CRI"
```

Comment:
Also, this needs to follow title casing per [our style guide](https://kubernetes.io/docs/contribute/style/style-guide/)

Reply:
```suggestion
title: "Kubernetes v1.31: Autoconfiguration For Node Cgroup Driver (beta)"
```